packages/cms-data-sync/src/labs/labs.data-migration.ts

```diff
@@ -0,0 +1,62 @@
+import { Labs } from '@asap-hub/contentful';
+import { labsQuery } from './labs.queries';
+import {
+  FetchLabsQuery,
+  FetchLabsQueryVariables,
+} from '../autogenerated-gql/graphql';
+import { getSquidexAndContentfulClients, logger } from '../utils';
+import { migrateFromSquidexToContentfulFactory } from '../utils/migration';
+
+type LabItem = NonNullable<
+  NonNullable<FetchLabsQuery['queryLabsContentsWithTotal']>['items']
+>[number];
+type ContentfulLab = Omit<Labs, 'sys' | 'contentfulMetadata'> & { id: string };
+
+export const migrateLabs = async () => {
+  const { contentfulEnvironment, squidexGraphqlClient } =
+    await getSquidexAndContentfulClients();
+
+  const migrateFromSquidexToContentful = migrateFromSquidexToContentfulFactory(
+    contentfulEnvironment,
+    logger,
+  );
+
+  const fetchAllData = async (skip: number = 0): Promise<LabItem[]> => {
```

Comment:
seems a little unnecessary considered we only have a few of those, especially since we don't paginate through teams for example, but fair enough. 

Why haven't you used to helper function you created though?

Reply:
I wasn't sure what the numbers in prod were like, but if it was up to me then all the content types would be resilient to pagination overflow.

Which helper function do you mean? There is one for fetching from Contentful, which I could re-purpose into a generic paginated fetch method.

Reply:
I had a look in production squidex and there are 258 labs listed, so these will need fetching in a paginated way.

There are also a little over 1,300 external authors, so these will also need to be refactored to be paginated.

Reply:
I've extracted a paginated fetch helper that can be repurposed across contentful and squidex queries, and applied it to external authors, I just need to rewrite the unit tests.